src/lib/storage/vector_compression/fixed_width_integer/fixed_width_integer_utils.hpp

```
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "storage/vector_compression/compressed_vector_type.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+inline bool is_fixed_width_integer(CompressedVectorType type) {
+  switch (type) {
+    case CompressedVectorType::FixedWidthInteger4Byte:
+    case CompressedVectorType::FixedWidthInteger2Byte:
+    case CompressedVectorType::FixedWidthInteger1Byte:
+      return true;
+    case CompressedVectorType::BitPacking:
+      return false;
+  }
+  return false;
```

Fail loudly (`Fail(<message>)`